pkg/kademlia/dialer.go

```diff
@@ -109,6 +109,23 @@ func (dialer *Dialer) FetchPeerIdentity(ctx context.Context, target pb.Node) (pI
 	return identity.PeerIdentityFromPeer(p)
 }
 
+// Info connects to a node address and returns its node info.
+func (dialer *Dialer) Info(ctx context.Context, address *pb.NodeAddress) (*pb.InfoResponse, error) {
+	if !dialer.limit.Lock() {
+		return nil, context.Canceled
+	}
+	defer dialer.limit.Unlock()
+
+	conn, err := dialer.dialAddress(ctx, address)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := conn.client.Info(ctx, &pb.InfoRequest{})
```

Comment:
Maybe `client.RequestInfo`? The GRPC methods seem to be verbs rather than nouns.

Reply:
Changed in latest commit.